src/libraries/Common/tests/CoreFx.Private.TestUtilities/System/TestEnvironment.cs

```diff
@@ -9,12 +9,12 @@ public static class TestEnvironment
         /// <summary>
         /// Check if the stress mode is enabled.
         /// </summary>
-        /// <value> true if the environment variable COREFX_STRESS set to '1' or 'true'. returns false otherwise</value>
+        /// <value> true if the environment variable DOTNET_TEST_STRESS set to '1' or 'true'. returns false otherwise</value>
```

Comment:
I can live with "DOTNET_TEST_*" But I would have preferred "DOTNETLIB_TEST_*".  I think "DOTNET" might be too generic and these variables are specifically about .NET Libraries stuff.

Reply:
Thanks, David. I don't have a strong preference here, but DOTNET_TEST_ seemed a nice correlary to where we use DOTNET_ for the product src. It's not clear to me there's an advantage to trying to make a distinction here between the VM and the libraries.

Let's go with what I have here. If it becomes an issue, we can certainly change it again.